src/tests/test_dynamic_api/test_views.py

```diff
@@ -565,3 +568,14 @@ def test_list_expand_true_for_loose_relation(
         response = api_client.get(url)
         assert response.status_code == 200, response.data
         assert response.data["_embedded"]["buurt"][0]["id"] == "03630000000078.2"
+
+
+@pytest.mark.django_db
+def test_wfs_view(api_client, filled_router):
+    wfs_url = (
+        "https://api.data.amsterdam.nl/v1/wfs/afvalwegingen/"
```

Comment:
Doet hij een externe request of steeds gebruikt ie local view?

Reply:
Dat klopt inderdaad niet. Copy paste foutje.